File: JAVA/Algoritmos/Primo.java
public class Primo {
    public int numero;
    public boolean TestePrimo(int numero) {
        this.numero = numero;
        if (numero == 1) {
            return false;
        }
        for (var i = 2; i <= Math.sqrt(numero); i++) {
            if (numero % i == 0) {
                return false;
            }
        }
        return true;
    }
}

File: javascript/Primo.js

function isPrime(number) {
    if (number <= 1) {
      console.log("Numero é menor ou igual a 1");
      return false;
    }
  
    for (let i = 2; i <= Math.sqrt(number); i++) {
      if (number % i === 0) {
        return false;
      }
    }
  
    console.log("Numero é primo");
    return true;
  }
